### ISSUE TITLE
I am a computer science student and I need to solve this problem

### ISSUE THREAD

**Issue by hishamshaker6** [pythonlessons/Keras-ResNet-tutorial#3]
![Screenshot (33)](https://user-images.githubusercontent.com/96210300/169386526-37b9c373-4ab1-442d-9fdd-80829c60829b.png)
model = Sequential()

model.add(tf.keras.layers.experimental.preprocessing.Rescaling(1./255))

model.add(Conv2D(filters=16, kernel_size=(3,3), padding='same', input_shape=(224, 224, 3)))

model.add(Activation('relu'))

model.add(BatchNormalization())

model.add(MaxPooling2D(pool_size=(2,2), strides=2, padding='valid'))

model.add(Conv2D(filters=32, kernel_size=(3,3), padding='same'))

model.add(Activation('relu'))

model.add(BatchNormalization())

model.add(MaxPooling2D(pool_size=(2,2), strides=2, padding='valid'))

model.add(Conv2D(filters=64, kernel_size=(3,3), padding='same'))

model.add(Activation('relu'))

model.add(BatchNormalization())

model.add(MaxPooling2D(pool_size=(2,2), strides=2, padding='valid'))

model.add(Flatten())

model.add(Dense(256))

model.add(Activation('relu'))

model.add(Dropout(0.3))

model.add(Dense(9))

model.add(Activation('softmax'))
